internal/rbd/controllerserver_test.go

```diff
@@ -67,3 +68,146 @@ func TestIsThickProvisionRequest(t *testing.T) {
 		t.Errorf("request should be for thick-provisioning: %s", req.Parameters["thickProvision"])
 	}
 }
+
+func TestIsMigrationVolHandle(t *testing.T) {
+	t.Parallel()
+	req := &csi.DeleteVolumeRequest{
+		VolumeId: "fake",
+	}
```

Comment:
why this is needed? use can use a variable?

Reply:
any wrong entry is good enough, just put it as a variable for now.